src/faebryk/exporters/pcb/cad/step.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of the faebryk project
+# SPDX-License-Identifier: MIT
+
+import logging
+import os
+import re
+import subprocess as sp
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+"""
+STEP file exporter using the kicad 8 cli
+
+
+"""
+
+
+def export_step(pcb_file: Path, path: Path):
+    logger.info(f"Exporting step file to {path}")
+    if not path.parent.exists():
+        os.makedirs(path.parent)
+
+    # check if kicad-cli 8 is installed
+    try:
+        sp.check_output("which kicad-cli", shell=True)
+    except sp.CalledProcessError:
+        raise Exception("kicad-cli is not installed")
+
+    # check if kicad-cli is version 8
+    try:
+        version = sp.check_output("kicad-cli --version", shell=True).decode("utf-8")
+        if not re.search(r"8\.\d+\.\d+", version):
```

Comment:
I think its enough to check whether its >= 8.0.0 (e.g 9.x.x should be also fine).